### PR TITLE
Update migrations

### PR DESCRIPTION
**Pull request by jechols**
The fields' help text values were previously set to binary "strings",
but the python 3 update changed that to a raw string, and django sees
these as different definitions.  The database schema won't actually be
changed, but future invocations of the `showmigrations` will be
confusing if we don't get this change in now.

### CODE REVIEW

**Comment by jechols** [comment]
FYI, I just now went back and did two full system wipes.  The first was followed by firing up the dev branch and exporting the DB schema straight from mariadb (`mysqldump -d -uopenoni -popenoni openoni`).  The second wipe was the same, but I ran it after this change.  The only differences in schema were timestamps of when the dump was run and the auto-increment number on the django_migrations table (since there's one extra migration that's been run).

tl;dr: absolutely positive these changes don't alter the schema.

**Comment by jduss4** [comment]
I applied this migration on top of my existing database on the python 3 branch and it looks good to me!  I don't know a lot about django migrations / schema generation / etc so hopefully I'm not overlooking something obvious.....!
